.jshintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "esnext": true,
+  "node": true
+}
```

Comment:
Ah, is there an eslint one that can be used instead? I think it'd be better to use the same rules/linter everywhere if possible.